### ISSUE TITLE
incompatibility with importlib_resources version 6

### ISSUE THREAD

**Issue by sebix** [laixintao/iredis#466]
With importlib_resources version 6.0.1 I get this stacktrace:
```
iredis/client.py:9: in <module>
    from importlib_resources import read_text
ImportError: cannot import name 'read_text' from 'importlib_resources' (/usr/lib/python3.11/site-packages/importlib_resources/__init__.py)
```

importlib_resources version 5.12.0 works fine

**Comment by laixintao** [comment]
hi sorry for the late, released in 1.13.2

**Comment by sebix** [comment]
Thanks for the fix! Package in openSUSE is updated. Will need to wait for the update of mistune, then the iredis package build will succeed again
